### DISCUSSION TITLE
What's the license of the docx files inside test/docx

### DISCUSSION THREAD

**Discussion by StephanMeijer** [jgm/pandoc#10232] [General]
Hey, I am currently writing a Pandoc alternative for OOXML/Docx. Our project is licensed under EUPL-1.2. At this moment, we are using `.docx` documents that are licensed under various Creative Commons licenses that allow their usage in our project as test documents.

What would be the license of the docx files inside [test/docx](https://github.com/jgm/pandoc/tree/main/test/docx)? I would love to use them.

**Comment by tarleb** [comment]
Unless noted otherwise in `COPYRIGHT`, all files are licensed under GPL-2.0-or-later. You might be interested in [this Q&A](https://opensource.stackexchange.com/q/10894/6254) that explains what that means for your use case. TL;DR: use the files, it's fine.

Excellent licensing choice BTW, I'm a big fan of the EUPL.

**Reply by StephanMeijer** [reply]
> Excellent licensing choice BTW, I'm a big fan of the EUPL.

Thanks, we are working for a government agency, so it's standard here!

**Comment by tarleb** [comment]
Is the code already available somewhere? Do you think it could somehow be made to work with pandoc in the future?

**Reply by StephanMeijer** [reply]
Certainly! https://gitlab.com/toegang-voor-iedereen/backend/nldoc-elixir/-/tree/feature/NLD-170-docx-conversion?ref_type=heads

You can see the logic here (work in progress :))

https://gitlab.com/toegang-voor-iedereen/backend/nldoc-elixir/-/tree/feature/NLD-170-docx-conversion/lib/conversion/docx?ref_type=heads

**Reply by StephanMeijer** [reply]
I am open for any collaboration or integration with Pandoc where it makes sense. We have a bit of a broader specification of documents which you can find here:
- https://gitlab.com/toegang-voor-iedereen/specification/document-specification
- https://spec-toegang-voor-iedereen-1f3acdcf76b75a137fedc6ad9a2cfabaab56.gitlab.io/

**Reply by tarleb** [reply]
Would you have time for a short (~30 min) call some time? I'd love to have a chat about that.
Please feel free to just send me an invite, my mail is in my profile.

**Reply by StephanMeijer** [reply]
Absolutely, I will reach out to you tomorrow or on Monday!
